tests/qos/test_voq_watchdog.py

```diff
@@ -35,19 +35,7 @@
 ]
 
 PKTS_NUM = 100
-
-
-@pytest.fixture(scope="function", autouse=True)
-def ignore_log_voq_watchdog(duthosts, loganalyzer):
-    if not loganalyzer:
-        yield
-        return
-    ignore_list = [r".*HARDWARE_WATCHDOG.*", r".*soft_reset*", r".*VOQ Appears to be stuck*"]
```

Comment:
Let's keep the 3 keys words, and only keep this change in the master branch, so that it can help to track the refactoring logic of sai.log->syslog.